test/second_q/algorithms/ground_state_solvers/test_groundstate_eigensolver.py

```diff
@@ -537,17 +538,67 @@ def test_vqe_ucc_factory_with_user_initial_point(self):
     def test_vqe_ucc_factory_with_mp2(self):
         """Test when using MP2InitialPoint to generate the initial point."""
 
-        informed_start = MP2InitialPoint()
+        initial_point = MP2InitialPoint()
 
         solver = VQEUCCFactory(
             quantum_instance=QuantumInstance(BasicAer.get_backend("statevector_simulator")),
-            initial_point=informed_start,
+            initial_point=initial_point,
+        )
+        calc = GroundStateEigensolver(self.qubit_converter, solver)
+        res = calc.solve(self.electronic_structure_problem)
+
+        np.testing.assert_array_almost_equal(
+            solver.initial_point.to_numpy_array(), self.mp2_initial_point
+        )
+        self.assertAlmostEqual(res.total_energies[0], self.reference_energy, places=6)
+
+    def test_vqe_ucc_factory_with_reps(self):
+        """Test when using MP2InitialPoint to generate the initial point with repeated evolved
```

Comment:
This test's docstring is misleading. As far as I can see, this test uses the default initial point which is not MP2 but hartree fock

Reply:
Good point. Thanks! Fixed.